crates/bevy_asset/src/diagnostic/asset_count_diagnostics_plugin.rs

```diff
@@ -29,9 +29,20 @@ impl<T: Asset> AssetCountDiagnosticsPlugin<T> {
     }
 
     pub fn setup_system(mut diagnostics: ResMut<Diagnostics>) {
+        let asset_type_name = std::any::type_name::<T>();
+        let max_length = MAX_DIAGNOSTIC_NAME_WIDTH - "asset_count ".len();
         diagnostics.add(Diagnostic::new(
             Self::diagnostic_id(),
-            format!("asset_count {}", std::any::type_name::<T>()),
+            format!(
+                "asset_count {}",
+                if asset_type_name.len() > max_length {
+                    asset_type_name
+                        .split_at(asset_type_name.len() - max_length + 1)
+                        .1
```

Comment:
```suggestion
                        .0
```
This should be the first `&str` returned by `split_at` (assuming you want the first part of the type name).

Reply:
I want the second, the first is most likely to be part of the path: `asset_count bevy_render::texture::texture::Texture`